### PR TITLE
Bump christophwurst/id3parser to 0.1.3

### PR DESCRIPTION
**Pull request by come-nc**
Signed-off-by: Côme Chilliet <come.chilliet@nextcloud.com>

Needed for PHP 8.1 compatibility

### CODE REVIEW

File: christophwurst/id3parser/src/getID3/getid3_lib.php [hunk 1]
```diff
@@ -1147,7 +1147,7 @@ public static function trimNullByte($string) {
 	* @param string $suffix If the name component ends in suffix this will also be cut off.
 	* @return string
 	*/
-	public static function mb_basename($path, $suffix = null) {
+	public static function mb_basename($path, $suffix = '') {
```
Comment (st3iny) [inline christophwurst/id3parser/src/getID3/getid3_lib.php:1150]: Why not fix it upstream at https://github.com/ChristophWurst/ID3Parser?

**Comment by st3iny** [comment]
Next release will also contain https://github.com/ChristophWurst/ID3Parser/pull/8 which should bring more improvements in this area.

**Comment by ChristophWurst** [comment]
I just released v0.1.4 with @st3iny's php8.1 fixes

File: christophwurst/id3parser/src/getID3/getid3_lib.php (hunk 1 above)
Comment (come-nc) [reply christophwurst/id3parser/src/getID3/getid3_lib.php:1150]: It is fixed upstream, this PR is just a composer require.
upstream fix is https://github.com/ChristophWurst/ID3Parser/commit/9c1d395fdcde3003bb99c36cc2f4d2b48a432871

**Pull request closed without merging**